README.md

```diff
@@ -1,5 +1,6 @@
 # ConstructR-Consul #
 
+[![Build Status](https://travis-ci.org/nikdon/constructr-consul.svg?branch=master)](https://travis-ci.org/nikdon/constructr-consul)
```

Comment:
This should be `/Tecsisa/constructr-consul`

Can you please change it? :) other than that LGTM